scripts/multiqc

```diff
@@ -459,10 +439,17 @@ plots_flat, plots_interactive, make_pdf, config_file, cl_config, verbose, quiet,
                 logger.warning("Deleting    : {}   (-f was specified)".format(os.path.relpath(config.output_fn)))
                 os.remove(config.output_fn)
             else:
-                logger.error("MultiQC Report {} already exists.".format(config.output_fn))
+                report_num = 0
                 logger.info("Use -f or --force to overwrite existing reports")
-                shutil.rmtree(tmp_dir)
-                sys.exit(1)
+                while os.path.exists(output_name):
```

Comment:
You haven't defined this variable yet, so I get the following error:

```python
Traceback (most recent call last):
  File "/Users/philewels/.miniconda/envs/work/bin/multiqc", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/philewels/GitHub/MultiQC/scripts/multiqc", line 629, in <module>
    multiqc()
  File "/Users/philewels/.miniconda/envs/work/lib/python2.7/site-packages/click/core.py", line 700, in __call__
    return self.main(*args, **kwargs)
  File "/Users/philewels/.miniconda/envs/work/lib/python2.7/site-packages/click/core.py", line 680, in main
    rv = self.invoke(ctx)
  File "/Users/philewels/.miniconda/envs/work/lib/python2.7/site-packages/click/core.py", line 873, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/philewels/.miniconda/envs/work/lib/python2.7/site-packages/click/core.py", line 508, in invoke
    return callback(*args, **kwargs)
  File "/Users/philewels/GitHub/MultiQC/scripts/multiqc", line 444, in multiqc
    while os.path.exists(output_name):
UnboundLocalError: local variable 'output_name' referenced before assignment
```

But anyway, I think a cleaner way to write this (without the intermediate variable `output_name` would be like this:

```python
while os.path.exists(config.output_fn):
    base, ext = os.path.splitext(config.output_fn_name)
    config.output_fn = os.path.join(config.output_dir, "{}_{}{}".format(base, report_num, ext) )
    report_num += 1
config.output_fn_name = os.path.basename(config.output_fn)
```
(`config.output_fn_name` wasn't being updated before and is used elsewhere)